contributing_to_docs/doc_guidelines.adoc

```diff
@@ -128,62 +128,41 @@ ID. For example:
-cases, create your concepts as individual modules and include them in
-appropriate assemblies. Avoid using gerunds in concept titles. "About <concept>"
-is a common concept module title.
-
-For more information about writing concepts, see the
-link:https://redhat-documentation.github.io/modular-docs/[Red Hat modular docs reference guide].
+== Writing assemblies
+An _assembly_ is a collection of modules that describes how to accomplish a user story.
 
-== Writing tasks
-A _task_ contains the steps that users follow to complete a process. Tasks
-contain ordered steps and explicit commands. In most cases, create your tasks
-as individual modules and include them in appropriate assemblies.
+For more information about forming assemblies, see the
+link:https://redhat-documentation.github.io/modular-docs/[Red Hat modular docs reference guide] and the link:https://github.com/redhat-documentation/modular-docs/blob/master/modular-docs-manual/files/TEMPLATE_ASSEMBLY_a-collection-of-modules.adoc[assembly template].
 
-[IMPORTANT]
+[NOTE]
 ====
-Use a verb in the task title (for example, Creating).
+For `Prerequistes`, `Next steps`, and `Additional resources` headings, use only `==` formatting in assembly files. For module files, use `.Prerequisites`, `.Next steps`, or `.Additional information` syntax to avoid TOC formatting within the module. Do not include xrefs in modules.
 ====
 
-Task modules take the following form:
-
-----
-<module_metadata>
-
-After the module file metadata, include a paragraph explaining why the user must
-perform this task. This should be 1-2 sentences maximum.
-
-If applicable, include any gotchas (things that could trip up the user or cause the task to fail).
-
-== Prerequisites
-
-* A bulleted list of prerequisites that MUST be performed before the user can complete this task.
-Skip if there isn't any related information.
-
-.Procedure
-
-. Step 1 - One command per step.
-
-. Step 2 - One command per step.
-
-. Step N
+== Writing concepts
+A _concept_ contains information to support the tasks that users want to do and
+must not include task information like commands or numbered steps. In most
+cases, create your concepts as individual modules and include them in
+appropriate assemblies.
 
-== Next steps
+Avoid using gerunds in concept titles. "About <concept>"
+is a common concept module title.
 
-You can explain any other tasks that MUST be completed after this task. You can
-skip this if there are none. Do not include xrefs. If the next steps are closely
-related to the task, you might be able to include their modules in the assembly.
+For more information about creating concept modules, see the
+link:https://redhat-documentation.github.io/modular-docs/[Red Hat modular docs reference guide] and the link:https://github.com/redhat-documentation/modular-docs/blob/master/modular-docs-manual/files/TEMPLATE_CONCEPT_concept-explanation.adoc[concept template].
 
-.Related information
+== Writing tasks
```

Comment:
Suggest replacing "task" with "procedure" throughout this section, since that's the terminology we use.